organize/models.py

```diff
@@ -28,20 +35,144 @@ class EventApplication(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
     # application fields
-    about_you = models.TextField()
-    why = models.TextField()
-    involvement = models.CharField(choices=INVOLVEMENT_CHOICES, max_length=15)
-    experience = models.TextField()
-    venue = models.TextField()
-    sponsorship = models.TextField()
-    coaches = models.TextField()
+    about_you = models.TextField("About organizer")
+    why = models.TextField("Motivations to organize")
+    involvement = models.CharField("Involvement in Django Girls",
+                                   choices=INVOLVEMENT_CHOICES, max_length=15)
+    experience = models.TextField("Experience with organizing other events")
+    venue = models.TextField("Information about your potential venue")
+    sponsorship = models.TextField("Information about your potential sponsorship")
+    coaches = models.TextField("Information about your potential coaches")
+
+    # status reflecting state of the event in a triaging process.
+    status = models.CharField(
+        choices=APPLICATION_STATUS,
+        default=NEW,
+        max_length=10,
+    )
+    status_changed_at = models.DateTimeField(null=True, blank=True)
+
+    comment = models.TextField(null=True, blank=True)
 
     class Meta:
         permissions = (
             ("can_accept_organize_application",
              "Can accept Organize Applications"),
         )
 
+    def __str__(self):
+        return "{}, {} ({})".format(self.city, self.country, self.get_status_display())
+
+    def create_event(self):
```

Comment:
Does this belong to the EventApplication model? This has nothing to do with the EventApplication itself, should be in the core app, I think :)